web-app/resources/java-worker/README.md

```diff
@@ -0,0 +1,76 @@
+# GoGoDuck
+
+GoGoDuck is a simple NetCDF file aggregator, using nco tools (ncks, ncrcat, ncatted, etc).
+
```

Comment:
So this replaces the shell script worker. Are you going to delete the .sh files then? I run the app and it works but what output am i seeing?